sunpy_sphinx_theme/sunpy/theme.conf

```diff
@@ -10,7 +10,8 @@ bootstrap_version = 3
 
 # Defaults for SunPy projects
 logo_url = https://sunpy.org/
-opengraph_image = img/sunpy_icon.svg
+# Opengraph does not support SVGs
+opengraph_image = sunpy_icon_128x128.png
```

Comment:
The issue is also that the path for this should be a full url which means adding this to the project's repo as well.